src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -265,18 +284,22 @@ export class HuiCardPicker extends LitElement {
         >
           ${!element || element.tagName === "HUI-ERROR-CARD"
             ? html`
-                ${this.hass!.localize(
-                  `ui.panel.lovelace.editor.card.${cardConfig.type}.description`
-                )}
+                ${customCard && customCard.description
+                  ? customCard.description
+                  : this.hass!.localize(
+                      `ui.panel.lovelace.editor.card.${cardConfig.type}.description`
+                    )}
               `
             : html`
                 ${element}
               `}
         </div>
         <div class="card-header">
-          ${this.hass!.localize(
-            `ui.panel.lovelace.editor.card.${cardConfig.type}.name`
-          )}
+          ${customCard && customCard.name
```

Comment:
```suggestion
          ${customCard?.name
```

Reply:
If there is no name in the custom card, we're going to translate the name? That will always be empty.